docs/remote_debug_setup_for_windows.rst

```diff
@@ -0,0 +1,110 @@
+Setup for Remote Debugging
+==========================
+
+for IntelliJ IDEA 13.1.6, Ultimate Edition on Windows
+
+Before Starting
+---------------
+
+-  It is assumed you have Treeherder setup in a VM already
+   [instructions] (installation\_for\_windows.rst) and can run both
+
+   -  ``./manage.py runserver``
+   -  ``celery -A treeherder worker -B --concurrency 5``
```

Comment:
The command line here might change. I'd point to the relevant section elsewhere in the documentation, instead of copying stuff here.